grpc/src/main/java/com/linecorp/armeria/client/grpc/GrpcClientOptions.java

```diff
@@ -99,15 +104,15 @@
      *       {@link MessageMarshaller.Builder#preservingProtoFieldNames(boolean)} via
      *       {@link GrpcJsonMarshallerBuilder#jsonMarshallerCustomizer(Consumer)}.
      *       <pre>{@code
-     *        Clients.builder(grpcServerUri)
-     *               .option(GrpcClientOptions.GRPC_JSON_MARSHALLER_FACTORY.newValue(serviceDescriptor -> {
-     *                   return GrpcJsonMarshaller.builder()
-     *                                            .jsonMarshallerCustomizer(builder -> {
-     *                                                builder.preservingProtoFieldNames(true);
-     *                                            })
-     *                                            .build(serviceDescriptor);
-     *               }))
-     *               .build();
+     *       Clients.builder(grpcServerUri)
+     *              .option(GrpcClientOptions.GRPC_JSON_MARSHALLER_FACTORY.newValue(serviceDescriptor -> {
+     *                  return GrpcJsonMarshaller.builder()
+     *                                           .jsonMarshallerCustomizer(builder -> {
+     *                                               builder.preservingProtoFieldNames(true);
+     *                                           })
+     *                                           .build(serviceDescriptor);
+     *              }))
+     *              .build();
```

Comment:
Thanks, @ikhoon :smile:

Reply:
Should we replace this example with the version that uses `GrpcClients`?

Reply:
Sounds good. Let me update it.